modules/installation-initializing.adoc

```diff
@@ -326,6 +326,13 @@ After you create the installation configuration file, you can modify the file to
 ====
 
 ... Select the default vCenter datastore to use.
++
+[WARNING]
+====
+You can specify the path of any datastore that exists in a datastore cluster. By default, Distributed Resource Scheduler (DRS), which uses Storage vMotion, is automatically enabled for a datastore cluster. Red Hat does not support Storage vMotion, so you must disable Storage DRS to avoid data loss issues for your {product-title} cluster.
```

Comment:
```suggestion
You can specify the path of any datastore that exists in a datastore cluster. By default, Storage Distributed Resource Scheduler (DRS), which uses Storage vMotion, is automatically enabled for a datastore cluster. Red Hat does not support Storage vMotion, so you must disable Storage DRS to avoid data loss issues for your {product-title} cluster.
```

Reply:
hanks, @jcpowermac . Updated.